README.md

```
@@ -56,20 +56,31 @@ delivery can be configured from. See the [log bucket](docs/log-bucket/), [log
+A `rainlytics` command ships beside them, and answers the questions people
+ask most without any SQL:
 
 ```bash
-npx @kensio/rainlytics query "SELECT cs_uri_stem, count(*) AS views
-  FROM cloudfront_logs
-  WHERE year = '2026' AND month = '08' AND day = '27'
-  GROUP BY 1 ORDER BY 2 DESC LIMIT 5"
+npx @kensio/rainlytics pageviews --last 7d
 ```
 
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)